ui-components/src/headers/Hero.tsx

```diff
@@ -15,11 +16,13 @@ export interface HeroProps {
   centered?: boolean
 }
 
-const HeroButton = (props: { title: string; href: string; className?: string }) => (
-  <span className={props.className + " hero__button"}>
-    <LinkButton href={props.href}>{props.title}</LinkButton>
-  </span>
-)
+const HeroButton = (props: { title: string; href: string; className?: string }) => {
+  return (
+    <a href={props.href} className={props.className + " hero__button"}>
+      {props.title}
+    </a>
+  )
+}
```

Comment:
Do you think all these changes are going to be upstream compatible or do we need to create our own component class?